### ISSUE TITLE
No commands will run

### ISSUE THREAD

**Issue by alex-taffe** [CocoaPods/CocoaPods#6345]
### Command

```
/usr/local/bin/pod install
```

### Report

* What did you do?
All commands fail to run except for clean
* What did you expect to happen?
The commands to run properly
* What happened instead?
I get the following crash

### Stack

```
   CocoaPods : 1.1.1
        Ruby : ruby 2.0.0p648 (2015-12-16 revision 53162) [universal.x86_64-darwin16]
    RubyGems : 2.0.14.1
        Host : Mac OS X 10.12.2 (16C63a)
       Xcode : 8.2.1 (8C1002)
         Git : git version 2.10.1 (Apple Git-78)
Ruby lib dir : /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib
Repositories : 0xced - https://github.com/0xced/XCDYouTubeKit.git @ 020a4aacb7df323947b3a092b63e7b416705bdd7
               master - https://github.com/CocoaPods/Specs.git @ 6e637cc691b7b0f6ec1df0d363c5e19223506d90
```

### Plugins

```
cocoapods-clean           : 0.0.1
cocoapods-deintegrate     : 1.0.1
cocoapods-plugins         : 1.0.0
cocoapods-search          : 1.0.0
cocoapods-stats           : 1.0.0
cocoapods-trunk           : 1.1.2
cocoapods-try             : 1.1.0
cocoapods-try-release-fix : 0.1.2
```

### Podfile

```ruby
platform :ios, '9.0'
use_frameworks!
target 'Broflo' do
	pod 'FacebookCore'
	pod 'FacebookLogin'
	pod 'FacebookShare'
	pod 'SwiftyJSON'
	pod 'MBProgressHUD'
	pod 'Eureka', '~>2.0.0-beta.1'
	pod 'iCarousel'
	pod 'SDWebImage'
	pod 'RSKImageCropper'
	pod 'MMUploadImage'
	pod 'Fabric'
	pod 'Crashlytics'
	pod 'SGAPI'
	pod 'FilterBar'
end
```

### Error

```
LoadError - cannot load such file -- cfpropertylist
/Library/Ruby/Gems/2.0.0/gems/xcodeproj-1.4.2/lib/xcodeproj/plist.rb:25:in `read_from_path'
/Library/Ruby/Gems/2.0.0/gems/xcodeproj-1.4.2/lib/xcodeproj/project.rb:200:in `initialize_from_file'
/Library/Ruby/Gems/2.0.0/gems/xcodeproj-1.4.2/lib/xcodeproj/project.rb:102:in `open'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-1.1.1/lib/cocoapods/installer/analyzer.rb:855:in `block (2 levels) in inspect_targets_to_integrate'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-1.1.1/lib/cocoapods/installer/analyzer.rb:854:in `each'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-1.1.1/lib/cocoapods/installer/analyzer.rb:854:in `block in inspect_targets_to_integrate'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-1.1.1/lib/cocoapods/user_interface.rb:64:in `section'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-1.1.1/lib/cocoapods/installer/analyzer.rb:849:in `inspect_targets_to_integrate'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-1.1.1/lib/cocoapods/installer/analyzer.rb:66:in `analyze'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-1.1.1/lib/cocoapods/installer.rb:236:in `analyze'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-1.1.1/lib/cocoapods/installer.rb:150:in `block in resolve_dependencies'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-1.1.1/lib/cocoapods/user_interface.rb:64:in `section'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-1.1.1/lib/cocoapods/installer.rb:149:in `resolve_dependencies'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-1.1.1/lib/cocoapods/installer.rb:110:in `install!'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-1.1.1/lib/cocoapods/command/install.rb:37:in `run'
/Library/Ruby/Gems/2.0.0/gems/claide-1.0.1/lib/claide/command.rb:334:in `run'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-1.1.1/lib/cocoapods/command.rb:50:in `run'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-1.1.1/bin/pod:55:in `<top (required)>'
/usr/local/bin/pod:23:in `load'
/usr/local/bin/pod:23:in `<main>'
```

**Comment by segiddins** [comment]
What happens if you run `gem install CFPropertyList`?

**Comment by alex-taffe** [comment]
@segiddins Same issue:

**Comment by mnmeyers** [comment]
I think I have the same issue. I ran `pod --version` and `pod init` both of which yielded the same crash report. Tried uninstalling and reinstalling. installed for the first time today.

```Process:               ruby [45428]
Path:                  /Users/USER/*/ruby
Identifier:            ruby
Version:               0
Code Type:             X86-64 (Native)
Parent Process:        zsh [1305]
Responsible:           iTerm [334]
User ID:               501

Date/Time:             2016-12-25 17:36:47.341 -0800
OS Version:            Mac OS X 10.11.6 (15G31)
Report Version:        11

Time Awake Since Boot: 440000 seconds
Time Since Wake:       6500 seconds

System Integrity Protection: enabled

Crashed Thread:        0  Dispatch queue: com.apple.main-thread

Exception Type:        EXC_BAD_ACCESS (SIGABRT)
Exception Codes:       KERN_INVALID_ADDRESS at 0x0000000000000418

VM Regions Near 0x418:
--> 
    __TEXT                 00000001001a9000-00000001001aa000 [    4K] r-x/rwx SM=COW  /Users/USER/*

Application Specific Information:
abort() called

Thread 0 Crashed:: Dispatch queue: com.apple.main-thread
0   libsystem_kernel.dylib        	0x00007fff87311f06 __pthread_kill + 10
1   libsystem_pthread.dylib       	0x00007fff971474ec pthread_kill + 90
2   libsystem_c.dylib             	0x00007fff93edb6df abort + 129
3   libruby.2.2.0.dylib           	0x00000001001ebfc9 die + 9
4   libruby.2.2.0.dylib           	0x00000001001ec1fa rb_bug_context + 554
5   libruby.2.2.0.dylib           	0x00000001002c2ad3 sigsegv + 83 (signal.c:794)
6   libsystem_platform.dylib      	0x00007fff95a9952a _sigtramp + 26
7   ???                           	000000000000000000 0 + 0
8   libruby.2.0.0.dylib           	0x0000000100bafb2f rb_newobj_of + 21
9   libruby.2.0.0.dylib           	0x0000000100c4cccd 0x100b71000 + 900301
10  libruby.2.0.0.dylib           	0x0000000100ba454a rb_require + 11
11  parser.bundle                 	0x0000000100b69897 Init_parser + 19 (parser.rl:886)
12  libruby.2.2.0.dylib           	0x00000001001ae04d dln_load + 221 (dln.c:1341)
13  libruby.2.2.0.dylib           	0x000000010033ada2 rb_vm_call_cfunc + 418 (.vm_insnhelper.c:110)
14  libruby.2.2.0.dylib           	0x00000001001f9983 rb_require_internal + 1587 (load.c:1010)
15  libruby.2.2.0.dylib           	0x00000001001f91f8 rb_f_require + 24 (load.c:1054)
16  libruby.2.2.0.dylib           	0x000000010033f46f vm_call_cfunc + 1551 (.vm_insnhelper.c:1382)
17  libruby.2.2.0.dylib           	0x000000010033e1a8 vm_call_method + 1016 (.vm_insnhelper.c:1691)
18  libruby.2.2.0.dylib           	0x0000000100328696 vm_exec_core + 13350 (insns.def:1054)
19  libruby.2.2.0.dylib           	0x0000000100339921 vm_exec + 129 (vm.c:1400)
20  libruby.2.2.0.dylib           	0x000000010033986d rb_iseq_eval + 285 (vm.c:1658)
21  libruby.2.2.0.dylib           	0x00000001001fa80b rb_load_internal0 + 315 (load.c:614)
22  libruby.2.2.0.dylib           	0x00000001001f98b7 rb_require_internal + 1383 (load.c:643)
23  libruby.2.2.0.dylib           	0x00000001001f91f8 rb_f_require + 24 (load.c:1054)
24  libruby.2.2.0.dylib           	0x000000010033f46f vm_call_cfunc + 1551 (.vm_insnhelper.c:1382)
25  libruby.2.2.0.dylib           	0x0000000100328696 vm_exec_core + 13350 (insns.def:1054)
26  libruby.2.2.0.dylib           	0x0000000100339921 vm_exec + 129 (vm.c:1400)
27  libruby.2.2.0.dylib           	0x000000010033986d rb_iseq_eval + 285 (vm.c:1658)
28  libruby.2.2.0.dylib           	0x00000001001fa80b rb_load_internal0 + 315 (load.c:614)
29  libruby.2.2.0.dylib           	0x00000001001f98b7 rb_require_internal + 1383 (load.c:643)
30  libruby.2.2.0.dylib           	0x00000001001f91f8 rb_f_require + 24 (load.c:1054)
31  libruby.2.2.0.dylib           	0x000000010033f46f vm_call_cfunc + 1551 (.vm_insnhelper.c:1382)
32  libruby.2.2.0.dylib           	0x000000010033e1a8 vm_call_method + 1016 (.vm_insnhelper.c:1691)
33  libruby.2.2.0.dylib           	0x0000000100328696 vm_exec_core + 13350 (insns.def:1054)
34  libruby.2.2.0.dylib           	0x0000000100339921 vm_exec + 129 (vm.c:1400)
35  libruby.2.2.0.dylib           	0x000000010033986d rb_iseq_eval + 285 (vm.c:1658)
36  libruby.2.2.0.dylib           	0x00000001001fa80b rb_load_internal0 + 315 (load.c:614)
37  libruby.2.2.0.dylib           	0x00000001001f98b7 rb_require_internal + 1383 (load.c:643)
38  libruby.2.2.0.dylib           	0x00000001001f91f8 rb_f_require + 24 (load.c:1054)
39  libruby.2.2.0.dylib           	0x000000010033f46f vm_call_cfunc + 1551 (.vm_insnhelper.c:1382)
40  libruby.2.2.0.dylib           	0x0000000100328696 vm_exec_core + 13350 (insns.def:1054)
41  libruby.2.2.0.dylib           	0x0000000100339921 vm_exec + 129 (vm.c:1400)
42  libruby.2.2.0.dylib           	0x000000010033986d rb_iseq_eval + 285 (vm.c:1658)
43  libruby.2.2.0.dylib           	0x00000001001fa80b rb_load_internal0 + 315 (load.c:614)
44  libruby.2.2.0.dylib           	0x00000001001f98b7 rb_require_internal + 1383 (load.c:643)
45  libruby.2.2.0.dylib           	0x00000001001f91f8 rb_f_require + 24 (load.c:1054)
46  libruby.2.2.0.dylib           	0x000000010033f46f vm_call_cfunc + 1551 (.vm_insnhelper.c:1382)
47  libruby.2.2.0.dylib           	0x0000000100328696 vm_exec_core + 13350 (insns.def:1054)
48  libruby.2.2.0.dylib           	0x0000000100339921 vm_exec + 129 (vm.c:1400)
49  libruby.2.2.0.dylib           	0x000000010033986d rb_iseq_eval + 285 (vm.c:1658)
50  libruby.2.2.0.dylib           	0x00000001001fa80b rb_load_internal0 + 315 (load.c:614)
51  libruby.2.2.0.dylib           	0x00000001001f98b7 rb_require_internal + 1383 (load.c:643)
52  libruby.2.2.0.dylib           	0x00000001001f91f8 rb_f_require + 24 (load.c:1054)
53  libruby.2.2.0.dylib           	0x000000010033f46f vm_call_cfunc + 1551 (.vm_insnhelper.c:1382)
54  libruby.2.2.0.dylib           	0x000000010033e1a8 vm_call_method + 1016 (.vm_insnhelper.c:1691)
55  libruby.2.2.0.dylib           	0x0000000100328696 vm_exec_core + 13350 (insns.def:1054)
56  libruby.2.2.0.dylib           	0x0000000100339921 vm_exec + 129 (vm.c:1400)
57  libruby.2.2.0.dylib           	0x000000010033986d rb_iseq_eval + 285 (vm.c:1658)
58  libruby.2.2.0.dylib           	0x00000001001fa80b rb_load_internal0 + 315 (load.c:614)
59  libruby.2.2.0.dylib           	0x00000001001f98b7 rb_require_internal + 1383 (load.c:643)
60  libruby.2.2.0.dylib           	0x00000001001f91f8 rb_f_require + 24 (load.c:1054)
61  libruby.2.2.0.dylib           	0x000000010033f46f vm_call_cfunc + 1551 (.vm_insnhelper.c:1382)
62  libruby.2.2.0.dylib           	0x000000010033e1a8 vm_call_method + 1016 (.vm_insnhelper.c:1691)
63  libruby.2.2.0.dylib           	0x0000000100328696 vm_exec_core + 13350 (insns.def:1054)
64  libruby.2.2.0.dylib           	0x0000000100339921 vm_exec + 129 (vm.c:1400)
65  libruby.2.2.0.dylib           	0x000000010033986d rb_iseq_eval + 285 (vm.c:1658)
66  libruby.2.2.0.dylib           	0x00000001001fa80b rb_load_internal0 + 315 (load.c:614)
67  libruby.2.2.0.dylib           	0x00000001001f98b7 rb_require_internal + 1383 (load.c:643)
68  libruby.2.2.0.dylib           	0x00000001001f91f8 rb_f_require + 24 (load.c:1054)
69  libruby.2.2.0.dylib           	0x000000010033f46f vm_call_cfunc + 1551 (.vm_insnhelper.c:1382)
70  libruby.2.2.0.dylib           	0x000000010033e1a8 vm_call_method + 1016 (.vm_insnhelper.c:1691)
71  libruby.2.2.0.dylib           	0x0000000100328696 vm_exec_core + 13350 (insns.def:1054)
72  libruby.2.2.0.dylib           	0x0000000100339921 vm_exec + 129 (vm.c:1400)
73  libruby.2.2.0.dylib           	0x000000010033986d rb_iseq_eval + 285 (vm.c:1658)
74  libruby.2.2.0.dylib           	0x00000001001fa80b rb_load_internal0 + 315 (load.c:614)
75  libruby.2.2.0.dylib           	0x00000001001fa03a rb_f_load + 186 (load.c:705)
76  libruby.2.2.0.dylib           	0x000000010033f46f vm_call_cfunc + 1551 (.vm_insnhelper.c:1382)
77  libruby.2.2.0.dylib           	0x000000010033e1a8 vm_call_method + 1016 (.vm_insnhelper.c:1691)
78  libruby.2.2.0.dylib           	0x0000000100328696 vm_exec_core + 13350 (insns.def:1054)
79  libruby.2.2.0.dylib           	0x0000000100339921 vm_exec + 129 (vm.c:1400)
80  libruby.2.2.0.dylib           	0x00000001003466e8 eval_string_with_cref + 1864 (.vm_eval.c:1328)
81  libruby.2.2.0.dylib           	0x0000000100336731 rb_f_eval + 161 (.vm_eval.c:1406)
82  libruby.2.2.0.dylib           	0x000000010033f46f vm_call_cfunc + 1551 (.vm_insnhelper.c:1382)
83  libruby.2.2.0.dylib           	0x000000010033e1a8 vm_call_method + 1016 (.vm_insnhelper.c:1691)
84  libruby.2.2.0.dylib           	0x0000000100328696 vm_exec_core + 13350 (insns.def:1054)
85  libruby.2.2.0.dylib           	0x0000000100339921 vm_exec + 129 (vm.c:1400)
86  libruby.2.2.0.dylib           	0x000000010033a9dd rb_iseq_eval_main + 509 (vm.c:1671)
87  libruby.2.2.0.dylib           	0x00000001001f5094 ruby_exec_internal + 148 (eval.c:257)
88  libruby.2.2.0.dylib           	0x00000001001f4fa6 ruby_run_node + 54 (eval.c:309)
89  ruby                          	0x00000001001a9f1f main + 79 (main.c:36)
90  libdyld.dylib                 	0x00007fff9649b5ad start + 1

Thread 1:: ruby-timer-thr
0   libsystem_kernel.dylib        	0x00007fff873132a2 poll + 10
1   libruby.2.2.0.dylib           	0x0000000100355f53 thread_timer + 371 (.thread_pthread.c:1439)
2   libsystem_pthread.dylib       	0x00007fff9714499d _pthread_body + 131
3   libsystem_pthread.dylib       	0x00007fff9714491a _pthread_start + 168
4   libsystem_pthread.dylib       	0x00007fff97142351 thread_start + 13

Thread 0 crashed with X86 Thread State (64-bit):
  rax: 0x0000000000000000  rbx: 0x0000000000000006  rcx: 0x00007f9f440eb208  rdx: 0x0000000000000000
  rdi: 0x0000000000000a13  rsi: 0x0000000000000006  rbp: 0x00007f9f440eb230  rsp: 0x00007f9f440eb208
   r8: 0x0000000000000040   r9: 0x00007fff76cd01e0  r10: 0x0000000008000000  r11: 0x0000000000000206
  r12: 0x00007f9f440eb350  r13: 0x0000000000000050  r14: 0x00007fff7757c000  r15: 0x0000000000000000
  rip: 0x00007fff87311f06  rfl: 0x0000000000000206  cr2: 0x00007fff76cce008
  
Logical CPU:     0
Error Code:      0x02000148
Trap Number:     133


Binary Images:
       0x1001a9000 -        0x1001a9fff +ruby (0) <A5EB6A1A-8A47-3471-97E6-2FD4D1C0D635> /Users/USER/*/ruby
       0x1001ad000 -        0x1003d4ff7 +libruby.2.2.0.dylib (0) <7D234AED-7C8A-3FE8-8EEE-70F90DA98C7B> /Users/USER/*/libruby.2.2.0.dylib
       0x1007c6000 -        0x1007c7ff7 +encdb.bundle (0) <1D10DB28-4ACF-374A-80C3-1F20FA9ABDFB> /Users/USER/*/encdb.bundle
       0x1007ca000 -        0x1007cbff7 +transdb.bundle (0) <2DD4252C-A8B1-30B5-A807-748A7BD6AA4D> /Users/USER/*/transdb.bundle
       0x1007ce000 -        0x1007d0ff7 +thread.bundle (0) <A19EC747-2865-3B5F-B8E4-F67F74A54A32> /Users/USER/*/thread.bundle
       0x1007d5000 -        0x1007d8fff +etc.bundle (0) <25E868A5-C6AE-381D-994E-8517F0DBA25A> /Users/USER/*/etc.bundle
       0x1007dd000 -        0x1007e2fff +pathname.bundle (0) <1D452809-17FE-3B72-AC45-DCC958A8A890> /Users/USER/*/pathname.bundle
       0x1007e9000 -        0x1007eaff7 +console.bundle (0) <8FAB20BA-911F-3BCE-97DF-ACECAF7985E7> /Users/USER/*/console.bundle
       0x1007ee000 -        0x100832ff7 +date_core.bundle (0) <68E3EA3F-6635-396C-B8A8-3C14DC38EA83> /Users/USER/*/date_core.bundle
       0x100850000 -        0x100861fff +bigdecimal.bundle (0) <BDC14247-94EC-3972-8111-3506E6BCC8B9> /Users/USER/*/bigdecimal.bundle
       0x100868000 -        0x100899fff +openssl.bundle (0) <9F825684-3211-3AD6-A527-BDEADC46985D> /Users/USER/*/openssl.bundle
       0x1008cc000 -        0x10090aff7 +libssl.1.0.0.dylib (0) <0624F8A6-D663-3336-8E9E-394645F40C92> /usr/local/opt/openssl/lib/libssl.1.0.0.dylib
       0x100927000 -        0x100a98a6f +libcrypto.1.0.0.dylib (0) <9DDCB4D1-F1F7-3996-A174-282BC1E18ECA> /usr/local/opt/openssl/lib/libcrypto.1.0.0.dylib
       0x100b10000 -        0x100b11ff7 +digest.bundle (0) <B5989B28-6BD8-3C34-9466-582C48277885> /Users/USER/*/digest.bundle
       0x100b16000 -        0x100b1afff +stringio.bundle (0) <E180527E-AAE8-3478-B2AF-E39145B03083> /Users/USER/*/stringio.bundle
       0x100b20000 -        0x100b20ff7 +fcntl.bundle (0) <982F385E-97B3-3179-953C-4BFFA092CC55> /Users/USER/*/fcntl.bundle
       0x100b23000 -        0x100b41ff7 +socket.bundle (0) <6E4D3AA0-2046-3452-A21E-56B0E622825E> /Users/USER/*/socket.bundle
       0x100b52000 -        0x100b5cff7 +zlib.bundle (0) <5EB53DB1-B32D-3384-823F-05A717D05E75> /Users/USER/*/zlib.bundle
       0x100b65000 -        0x100b66ff7 +windows_31j.bundle (0) <A6080874-2CE0-33CC-A8C5-0EA2D96D0CF8> /Users/USER/*/windows_31j.bundle
       0x100b69000 -        0x100b6cff7 +parser.bundle (0) <B748E3B8-5B67-3B17-95CE-2F7D7D04BB1E> /Users/USER/*/parser.bundle
       0x100b71000 -        0x100d19ff7  libruby.2.0.0.dylib (113.40.1) <00E2A76D-12F7-3B37-B601-5BC7EE289AAC> /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/libruby.2.0.0.dylib
    0x7fff63041000 -     0x7fff63078a47  dyld (360.22) <DC81CC9D-651A-3A45-8809-928282052BD3> /usr/lib/dyld
    0x7fff83fbd000 -     0x7fff83fc2ff7  libmacho.dylib (875.1) <318264FA-58F1-39D8-8285-1F6254EE410E> /usr/lib/system/libmacho.dylib
    0x7fff8402b000 -     0x7fff8405affb  libsystem_m.dylib (3105) <08E1A4B2-6448-3DFE-A58C-ACC7335BE7E4> /usr/lib/system/libsystem_m.dylib
    0x7fff84d1b000 -     0x7fff84d37ff7  libsystem_malloc.dylib (67.40.1) <5748E8B2-F81C-34C6-8B13-456213127678> /usr/lib/system/libsystem_malloc.dylib
    0x7fff84e2d000 -     0x7fff84e3eff7  libsystem_trace.dylib (201.10.3) <1DD0C1FD-74E2-3949-9C2C-B81D19C2CA5E> /usr/lib/system/libsystem_trace.dylib
    0x7fff854f2000 -     0x7fff854fbff3  libsystem_notify.dylib (150.40.1) <D48BDE34-0F7E-34CA-A0FF-C578E39987CC> /usr/lib/system/libsystem_notify.dylib
    0x7fff85c86000 -     0x7fff85ff1657  libobjc.A.dylib (680) <D55D5807-1FBE-32A5-9105-44D7AFE68C27> /usr/lib/libobjc.A.dylib
    0x7fff87236000 -     0x7fff87237fff  libsystem_secinit.dylib (20) <32B1A8C6-DC84-3F4F-B8CE-9A52B47C3E6B> /usr/lib/system/libsystem_secinit.dylib
    0x7fff872fb000 -     0x7fff87319ff7  libsystem_kernel.dylib (3248.60.10) <C1A6A0B9-1869-36AB-B4A2-D862EB09A4BE> /usr/lib/system/libsystem_kernel.dylib
    0x7fff873c9000 -     0x7fff8783ffff  com.apple.CoreFoundation (6.9 - 1258.1) <943A1383-DA6A-3DC0-ABCD-D9AEB3D0D34D> /System/Library/Frameworks/CoreFoundation.framework/Versions/A/CoreFoundation
    0x7fff87af3000 -     0x7fff87b0aff7  libsystem_coretls.dylib (83.40.5) <C90DAE38-4082-381C-A185-2A6A8B677628> /usr/lib/system/libsystem_coretls.dylib
    0x7fff880ea000 -     0x7fff8813dff7  libc++.1.dylib (120.1) <8FC3D139-8055-3498-9AC5-6467CB7F4D14> /usr/lib/libc++.1.dylib
    0x7fff88204000 -     0x7fff88204ff7  libunc.dylib (29) <DDB1E947-C775-33B8-B461-63E5EB698F0E> /usr/lib/system/libunc.dylib
    0x7fff896b8000 -     0x7fff896bdff3  libunwind.dylib (35.3) <F6EB48E5-4D12-359A-AB54-C937FBBE9043> /usr/lib/system/libunwind.dylib
    0x7fff8a6a3000 -     0x7fff8a8b0fff  libicucore.A.dylib (551.51.4) <3899B146-3840-3D4A-8C4A-FE391D5D25C7> /usr/lib/libicucore.A.dylib
    0x7fff8a8b6000 -     0x7fff8a8beffb  libsystem_dnssd.dylib (625.60.4) <80189998-32B0-316C-B5C5-53857486713D> /usr/lib/system/libsystem_dnssd.dylib
    0x7fff8b9b1000 -     0x7fff8b9b4fff  libsystem_sandbox.dylib (460.60.2) <2A68B39C-B786-3A05-87A2-56E688469FB8> /usr/lib/system/libsystem_sandbox.dylib
    0x7fff8b9c1000 -     0x7fff8b9eaff7  libxpc.dylib (765.50.8) <54D1328E-054E-3DAA-89E2-375722F9D18F> /usr/lib/system/libxpc.dylib
    0x7fff8c2a1000 -     0x7fff8c2a2ffb  libSystem.B.dylib (1226.10.1) <ECAA07BE-A114-3461-AA13-C57838775FC2> /usr/lib/libSystem.B.dylib
    0x7fff8d835000 -     0x7fff8d835ff7  libkeymgr.dylib (28) <8371CE54-5FDD-3CE9-B3DF-E98C761B6FE0> /usr/lib/system/libkeymgr.dylib
    0x7fff8e389000 -     0x7fff8e38dfff  libcache.dylib (75) <9548AAE9-2AB7-3525-9ECE-A2A7C4688447> /usr/lib/system/libcache.dylib
    0x7fff8ef15000 -     0x7fff8ef5bff7  libauto.dylib (186) <999E610F-41FC-32A3-ADCA-5EC049B65DFB> /usr/lib/libauto.dylib
    0x7fff8ef79000 -     0x7fff8ef80ff7  libcompiler_rt.dylib (62) <A13ECF69-F59F-38AE-8609-7B731450FBCD> /usr/lib/system/libcompiler_rt.dylib
    0x7fff8fc7e000 -     0x7fff8fc7fffb  libremovefile.dylib (41) <552EF39E-14D7-363E-9059-4565AC2F894E> /usr/lib/system/libremovefile.dylib
    0x7fff917f6000 -     0x7fff91820ff7  libc++abi.dylib (307.2) <922EFB36-0E9E-315B-8270-E81AC43472C0> /usr/lib/libc++abi.dylib
    0x7fff92233000 -     0x7fff9224aff7  libsystem_asl.dylib (323.50.1) <41F8E11F-1BD0-3F1D-BA3A-AA1577ED98A9> /usr/lib/system/libsystem_asl.dylib
    0x7fff93dca000 -     0x7fff93dcbfff  libsystem_blocks.dylib (65) <1244D9D5-F6AA-35BB-B307-86851C24B8E5> /usr/lib/system/libsystem_blocks.dylib
    0x7fff93e7d000 -     0x7fff93f0afef  libsystem_c.dylib (1082.60.1) <28733D22-553E-3CBC-8D2C-EDCEB46E46AF> /usr/lib/system/libsystem_c.dylib
    0x7fff9426e000 -     0x7fff9426eff7  liblaunch.dylib (765.50.8) <834ED605-5114-3641-AA4D-ECF31B801C50> /usr/lib/system/liblaunch.dylib
    0x7fff95a97000 -     0x7fff95a9ffef  libsystem_platform.dylib (74.40.2) <29A905EF-6777-3C33-82B0-6C3A88C4BA15> /usr/lib/system/libsystem_platform.dylib
    0x7fff963b4000 -     0x7fff9641aff7  libsystem_network.dylib (583.50.1) <B52DAB73-92DC-3DA7-B9F4-B899D66445C1> /usr/lib/system/libsystem_network.dylib
    0x7fff96486000 -     0x7fff96497ff7  libz.1.dylib (61.20.1) <B3EBB42F-48E3-3287-9F0D-308E04D407AC> /usr/lib/libz.1.dylib
    0x7fff96498000 -     0x7fff9649bffb  libdyld.dylib (360.22) <1F9FCF2C-5E94-3F2D-B4A5-1238587AA5C6> /usr/lib/system/libdyld.dylib
    0x7fff968e5000 -     0x7fff968edfff  libcopyfile.dylib (127) <A48637BC-F3F2-34F2-BB68-4C65FD012832> /usr/lib/system/libcopyfile.dylib
    0x7fff97135000 -     0x7fff9713dfff  libsystem_networkextension.dylib (385.40.36) <66095DC7-6539-38F2-95EE-458F15F6D014> /usr/lib/system/libsystem_networkextension.dylib
    0x7fff97141000 -     0x7fff9714aff7  libsystem_pthread.dylib (138.10.4) <3DD1EF4C-1D1B-3ABF-8CC6-B3B1CEEE9559> /usr/lib/system/libsystem_pthread.dylib
    0x7fff973a3000 -     0x7fff9741afeb  libcorecrypto.dylib (335.50.1) <B5C05FD7-A540-345A-87BF-8E41848A3C17> /usr/lib/system/libcorecrypto.dylib
    0x7fff97b8c000 -     0x7fff97b8dfff  libDiagnosticMessagesClient.dylib (100) <4243B6B4-21E9-355B-9C5A-95A216233B96> /usr/lib/libDiagnosticMessagesClient.dylib
    0x7fff97cda000 -     0x7fff97d07fff  libdispatch.dylib (501.40.12) <C7499857-61A5-3D7D-A5EA-65DCC8C3DF92> /usr/lib/system/libdispatch.dylib
    0x7fff97d0e000 -     0x7fff97d10ff7  libquarantine.dylib (80) <0F4169F0-0C84-3A25-B3AE-E47B3586D908> /usr/lib/system/libquarantine.dylib
    0x7fff97faa000 -     0x7fff97fb5ff7  libcommonCrypto.dylib (60075.50.1) <93732261-34B4-3914-B7A2-90A81A182DBA> /usr/lib/system/libcommonCrypto.dylib
    0x7fff985f6000 -     0x7fff9861ffff  libsystem_info.dylib (477.50.4) <FAA9226D-64DE-3769-A6D8-6CABA4B7FF4D> /usr/lib/system/libsystem_info.dylib
    0x7fff9895f000 -     0x7fff98961fff  libsystem_coreservices.dylib (19.2) <1B3F5AFC-FFCD-3ECB-8B9A-5538366FB20D> /usr/lib/system/libsystem_coreservices.dylib
    0x7fff98b61000 -     0x7fff98b63ff7  libsystem_configuration.dylib (802.40.13) <3DEB7DF9-6804-37E1-BC83-0166882FF0FF> /usr/lib/system/libsystem_configuration.dylib

External Modification Summary:
  Calls made by other processes targeting this process:
    task_for_pid: 0
    thread_create: 0
    thread_set_state: 0
  Calls made by this process:
    task_for_pid: 0
    thread_create: 0
    thread_set_state: 0
  Calls made by all processes on this machine:
    task_for_pid: 229311
    thread_create: 0
    thread_set_state: 39

VM Region Summary:
ReadOnly portion of Libraries: Total=113.6M resident=0K(0%) swapped_out_or_unallocated=113.6M(100%)
Writable regions: Total=56.6M written=0K(0%) resident=0K(0%) swapped_out=0K(0%) unallocated=56.6M(100%)
 
                                VIRTUAL   REGION 
REGION TYPE                        SIZE    COUNT (non-coalesced) 
===========                     =======  ======= 
Activity Tracing                  2048K        2 
Kernel Alloc Once                    4K        2 
MALLOC                            46.2M       12 
MALLOC guard page                   16K        4 
STACK GUARD                          4K        2 
Stack                             8216K        3 
Stack Guard                       56.0M        2 
VM_ALLOCATE                          4K        2 
__DATA                            3384K       70 
__LINKEDIT                        93.2M       25 
__TEXT                            20.4M       66 
__UNICODE                          552K        2 
shared memory                       12K        4 
===========                     =======  ======= 
TOTAL                            229.7M      183 
```

**Comment by cicerohellmann** [comment]
Same issue of the first guy!

**Comment by alex-taffe** [comment]
@mnmeyers interestingly, `pod --version` works for me

**Comment by mnmeyers** [comment]
hm so maybe it's a different problem and I should raise a separate issue

**Comment by alex-taffe** [comment]
@cicerohellmann @segiddins I managed to fix it. I ran `gem list` which yielded this as a result:
`…
CFPropertyList (2.3.4, 2.3.3, 2.2.8)
…`

I then ran `sudo gem cleanup CFPropertyList` which auto removed version 2.3.3 of CFPropertyList. I then modified `/Library/Ruby/Gems/2.0.0/gems/xcodeproj-1.4.2/lib/xcodeproj/plist.rb`line 2 from:
`autoload :CFPropertyList, ‘cfpropertylist'`
to
`autoload :CFPropertyList, ‘CFPropertyList’`

and CocoaPods loaded properly. I can create a pull request for this but I feel like it’d break other people’s installs. Not sure if this is specific to a few machines or if it is a new issue with 10.12.2

**Comment by alex-taffe** [comment]
@mnmeyers did my above comment fix your problem too?

**Comment by segiddins** [comment]
Ah do you have a case sensitive file system?

**Comment by alex-taffe** [comment]
@segiddins I do not, but I actually just reverted that ruby file to the one installed with the gem and it still worked. So I’m assuming it actually is only the multiple versions of CFPropertyList being installed? I don’t know much of anything about Ruby but it seems like that shouldn’t be an issue

**Comment by mnmeyers** [comment]
@alex-taffe when I run `gem list` I only have one version of `CFPropertyList` (v. 2.3.4) so I don't think that fix would make a difference for my problem. I'll create a new issue

**Comment by cicerohellmann** [comment]
This doesn't exist: /Library/Ruby/Gems/2.0.0/gems/xcodeproj-1.4.2/lib/xcodeproj/plist.rb 
And I have another thing, I just need some one to guide how to report here the clue I found. This mess just happens when I'm in  a branch that I installed Calabash! Seriously, when i switch back to the old branch, everything works. So, if some one is interested, calabash-ios setup can reproduce this issue.
